node/src/effect.rs

```diff
@@ -945,6 +945,24 @@ impl<REv> EffectBuilder<REv> {
         .await
     }
 
+    /// Puts a trie into the trie store and asynchronously returns any missing descendant trie keys.
```

Comment:
> Puts a trie into the trie store

Does it? Wouldn't it fail if `trie_key` isn't stored yet?

Reply:
I believe if `trie_key` isn't present it will just output `vec![trie_key]`.

See the [`missing_trie_keys`][1] implementation.

[1]: https://github.com/xcthulhu/casper-node/blob/afa4a647170b3c161cfa706919d1f5132c8ea7b6/execution_engine/src/storage/trie_store/operations/mod.rs#L235

Reply:
Makes sense. But then this documentation is still wrong, is it? It never puts anything into the trie store?

Reply:
You are right, the documentation is wrong.